.env.dist

```diff
@@ -1,9 +1,12 @@
 USER_ID=
```

Comment:
I think `USER_ID` and `PORT` are unused at this point. Let's get rid of them.

Reply:
Removed.